### PR TITLE
Documentation corrections

### PR DESCRIPTION
**Pull request by speller26**
* Correct "principle square root" to "principal square root"
* Correct the dagger gate docs to read "Adjoint of the principal square root"

### CODE REVIEW

**Comment by speller26** [comment]
@googlebot I signed it!

File: src/circuit/gate_data_period_4.cc [hunk 1]
```diff
@@ -144,7 +144,7 @@ H 0
                 return {
                     "B_Single Qubit Clifford Gates",
                     R"MARKDOWN(
-Principle square root of Y gate.
+Adjoint of the principal square root of Y gate.
```
Comment (Strilanc) [inline src/circuit/gate_data_period_4.cc:147]: This isn't the adjoint.

Comment (Strilanc) [inline src/circuit/gate_data_period_4.cc:147]: Also update SQRT_X_DAG's documentation.

**Review by Strilanc — CHANGES_REQUESTED** [review]

**Comment by Strilanc** [comment]
Thanks for the typo fix. I always get that particular pair of homonyms mixed up. There's just a change to SQRT_X_DAG missing.

File: src/circuit/gate_data_period_4.cc (hunk 1 above)
Comment (speller26) [reply src/circuit/gate_data_period_4.cc:147]: Done!

**Review by Strilanc — APPROVED** [review]

**Pull request merged**